docs/content/references/cli.mdx

```diff
@@ -36,5 +36,68 @@ The six most useful commands to users are the following:
 - **[IOTA Validator CLI](./cli/validator.mdx):** Use the `iota validator` command to access tools useful for IOTA validators.
 - **[IOTA Genesis Ceremony CLI](./cli/ceremony.mdx):** Use the `iota ceremony` command to build a genesis blob.
 
-
 Use the `help` flag for the commands that are not documented yet. For example, `iota validator --help`.  
+
+## IOTA CLI Completion
+
+<Tabs groupId="shell">
+
+<TabItem label="Bash" value="bash">
+
+```sh
+iota generate-completions --shell bash > ~/.iota/completion.bash
+echo "source ~/.iota/completion.bash" >> ~/.bashrc
+```
+
+</TabItem>
+
+<TabItem label="Elvish" value="elvish">
+
+TODO
+
+```sh
+iota generate-completions --shell elvish
+```
+
+</TabItem>
+
+<TabItem label="Fish" value="fish">
+
+```sh
+iota generate-completions --shell fish > iota.fish
+mv iota.fish ~/.config/fish/completions
+```
+
+</TabItem>
+
+<TabItem label="Nushell" value="nushell">
+
+TODO
+
+```sh
+iota generate-completions --shell nushell
+```
+
+</TabItem>
+
+<TabItem label="Powershell" value="powershell">
+
+TODO
+
+```sh
+iota generate-completions --shell powershell
+```
+
+</TabItem>
+
+<TabItem label="Zsh" value="zsh">
+
+```sh
+iota generate-completions --shell zsh > _iota
+sudo mv _iota /usr/share/zsh/functions/Completion/Zsh/
```

Comment:
Is there maybe another folder zsh looks for in `~/` so we don't need privileged rights?

Reply:
Haven't found something, one can create a new folder like `~/.zsh/completions` (could by anything, zsh doesn't look there automatically) and add it to .zshrc like this
```
fpath=(~/.zsh/completions $fpath)
autoload -Uz compinit
compinit
```
but that seems to be much more complicated